travis.sh

```diff
@@ -26,6 +26,9 @@ export TARGET_REPO_NAME=${TRAVIS_REPO_SLUG##*/}
 export PYTHONUNBUFFERED=${PYTHONUNBUFFERED:1}
 export _FOLDING_TYPE=travis
 
+# Update libseccomp to allow statx syscalls
+(sudo apt update && sudo apt install -y libseccomp2) &> "$(mktemp)"
```

Comment:
```suggestion
sudo apt-get -qq install -y libseccomp2"
```

Reply:
Not all versions of apt-get accept "-qq" after install, if I recall it correctly

Reply:
Not that I know of. It is included inside the man page tho.